core/modules/user/user.module

```diff
@@ -1630,6 +1631,19 @@ function user_menu_alter(&$items) {
   }
 }
 
+/**
+ * Return the user page tabs setting.
+ */
+function user_login_page_tabs() {
+  static $user_login_page_tabs;
+
+  if (!isset($user_login_page_tabs)) {
+    $user_login_page_tabs = config_get('system.core', 'user_login_page_tabs');
+  }
+
+  return $user_login_page_tabs;
```

Comment:
Seems like the only purpose of this function is to return a single config item.
What was the intention behind making this a full function? What's the benefit?
Config *is* already cached, AFAIK.
At first sight, this appears like over-abstraction to me, but I might miss something. :thinking:

Reply:
Thanks for pointing this out. So generally no, config isn't cached, except if the `_config_static` is true in the config file, which it is in the `system.core.json` file. So you're right we actually don't need this function in this case.